Zend/zend_vm_gen.php

```diff
@@ -19,7 +19,7 @@
 	 $Id$
 */
 
-$header_text = <<< DATA
+const HEADER_TEXT = <<< DATA
```

Comment:
I dont think you can assume that this script will be invoked with the php binary the git clone delivers.

I can think of some good arguments why a user wants to use a stable and released php binary to invoke this script.